app/assets/stylesheets/_entries.scss

```diff
@@ -128,6 +128,24 @@
 }
 
 .entries-table {
+  table-layout: auto;
+  line-height: 20px;
+
+  td,
+  th {
+    padding: 4px 4px;
```

Comment:
Properties should be ordered overflow, padding, text-overflow, white-space<br>Shorthand form for property `padding` should be written more concisely as `4px` instead of `4px 4px`